.reapack-index.conf

```diff
@@ -3,3 +3,5 @@
 --commit-template "index: $changelog
 
 [ci skip]"
+
+--ignore "MIDI/talagan_MIDI CC Mapper X/"
```

Comment:
Atm, they do, but that was for safety, for future releases. Since that new "extension" feature for MIDI CC Mapper X is already quite large (>70 files), I thought it could be a good idea to wildcard the full directory (that's the intended purpose, in fact) - all the more that there are already exotic file exts in that directory (.rb, .md, etc).  I let you decide what's the cleaner, since it's your house first :-)